src/events/cd-user-ticket-list-item.vue

```diff
@@ -0,0 +1,295 @@
+<template>
```

Comment:
I know you already mentioned this, but there are a lot of parallels between this file and cd-event-list-item. I'd definitely rather we look at how to share as much code as possible. I guess the main common items are the border, header, time and recurring notice.

Reply:
The border is used even more. Could be just a global style that can be used, or a really dumb component that encapsulates the style.